src/Raven.Server/ServerWide/Commands/Indexes/SetIndexLockCommand.cs

```diff
@@ -26,10 +26,17 @@ public SetIndexLockCommand(string name, IndexLockMode mode, string databaseName)
         public override void UpdateDatabaseRecord(DatabaseRecord record, long etag)
         {
             if (record.Indexes.TryGetValue(IndexName, out IndexDefinition staticIndex))
+            {
                 staticIndex.LockMode = LockMode;
+                staticIndex.Etag = etag;
```

Comment:
@maximburyak @ayende if the previous lock mode is the same as it was maybe this should be a no-op and the record should not be updated?